pages/team.php

```diff
@@ -409,7 +409,8 @@ function team() {
             $leader_avatar = $result['avatar_link'];
             $leader_avatar_size = User::AVATAR_MAX_SIZE;
             if($result['forbidden_seal']) {
-                $leader_avatar_size = User::AVATAR_MAX_SEAL_SIZE;
+                $result['forbidden_seal'] = json_decode($result['forbidden_seal'], true);
```

Comment:
Maybe it would be a helpful abstraction if we made a `ForbiddenSeal` class with a `ForbiddenSeal::fromDb` static method that accepts the value `$result['forbidden_seal']` provides, json decodes it, and populates it into a structured class. That would also allow us to do things like 

```php
$forbidden_seal = ForbiddenSeal::fromDb($result['forbidden_seal']);
$leader_avatar_size = $forbidden_seal->avatar_size;

// or

$player->forbidden_seal->avatar_size
```

Internally in the class we'd power it with this:
```php
public static function fromDb(string $raw_data) {
    $data = json_decode($raw_data, true);
    $this->level = $data['level'];

    // ...
    $this->avatar_size = System::$premium_benefits[$this->level]['avatar_size'];
    // ...
}
```